README.md

```diff
@@ -182,8 +182,10 @@ echo -1 | sudo tee /proc/sys/kernel/perf_event_paranoid
 ### DTrace on macOS
 
 On macOS, there is no alternative to running as superuser in order to
-enable dtrace. The simplest way is to use `--root`, this way `rustc`
-will be run normally but the binary will get run as superuser.
+enable DTrace. This should be done by invoking `cargo flamegraph --root ...`.
```

Comment:
I think this is overindexing on using `cargo flamegraph`, when just `flamegraph` is also perfectly viable.

Reply:
Updated!